chat_bot/prompts.py

```diff
@@ -7,22 +7,22 @@
     "sustainability": {
         "transport": "This is a conversation between two friends about the environment and transport.\n\nFriend_q: Do you take public transport often?\nYou: I try to, but it's not always possible with my work schedule.\nFriend_q: Do you live close to where you work?\nYou: Yes, I'm only a few blocks away. Do you cycle?\nFriend_a: I do cycle sometimes, but the weather can be a bit unpredictable.\nYou: Well, that's good. You're already doing your part by living close to where you work. What do you think about carbon footprints?\nFriend_a: It is so important to reduce our carbon footprint these days.",
         "recycling": "This is a conversation between two friends about recycling.\n\nFriend_q: Have you been recycling lately?\nYou: Yeah, I've been trying to do my part to help the environment.\nFriend_q: What kind of things do you recycle?\nYou: I recycle paper, plastic, and metal. What about you?\nFriend_a: I recycle paper, plastic, and metal too.\nYou: How frequently do you do it?\nFriend_a: I usually recycle every other week.",
-        "food_waste": "This is a conversation between two friends about food and the environment.\n\nFriend_q: What can we do about it food waste?\nYou: We can redistribute food to soup kitchens and food banks. Even restaurants have a lot of waste.\nFriend_q: What can we do about that?\nYou: We can ask them to donate their leftover food too. Why is there so much food waste on our planet?\nFriend_a: We have so much food, but we don't eat it all.\nYou: How come?\nFriend_a: Supermarkets have so much spare food they don't sell."
+        "food waste": "This is a conversation between two friends about food and the environment.\n\nFriend_q: What can we do about it food waste?\nYou: We can redistribute food to soup kitchens and food banks. Even restaurants have a lot of waste.\nFriend_q: What can we do about that?\nYou: We can ask them to donate their leftover food too. Why is there so much food waste on our planet?\nFriend_a: We have so much food, but we don't eat it all.\nYou: How come?\nFriend_a: Supermarkets have so much spare food they don't sell."
```

Comment:
Typo in Friend Q's first response is annoying me: "What can we do about it food waste?"